app.model/app.model.js

```diff
@@ -43,3 +43,14 @@ exports.fetchCommentsByArticleId = (articleId) => {
       return comments.rows;
     });
 };
+exports.addCommentToDB = (userData, articleId) => {
+  const { username, body } = userData;
```

Comment:
We could do this in the controller and pass the properties already destructured 🙂